web-common/src/components/data-types/PercentageChange.svelte

```diff
@@ -45,7 +57,7 @@
   {dark}
 >
   <slot name="value">
-    {#if isNoData}
+    {#if isNoData || isPercDiff(value)}
```

Comment:
Usage of `|| isPercDiff(value)` is redundant as `isNoData` already factors it. (Also applicable for line 22)